### ISSUE TITLE
Expose JMX monitoring for event queue size in Java client

### ISSUE THREAD

**Issue by wneild** [hazelcast/hazelcast#9130]
Similar to that which is available in the server, it would be incredibly useful to have a view of the event queue size in the client.

**Comment by bacsog** [comment]
+1

**Comment by longtimeago** [comment]
Should this approach http://docs.hazelcast.org/docs/latest-development/manual/html/Management/Monitoring_with_JMX.html
work for java client monitoring?

**Comment by pveentjer** [comment]
Client and server have diagnostics that dumps a huge quantity of data, including diagnostics data. Any probes missing like wan queue size can easily be added; if they are not yet already added.

@mmedenjak do you know which WAN probes are added to metrics?

Apart from that we already have a WAN diagnostics plugin; so perhaps this whole task can be closed?

**Comment by mmedenjak** [comment]
@pveentjer I don't think this issue is for WAN event queue, it is for the regular event queue (map, cache, ...). Currently we expose WAN queue size through a diagnostics plugin (amongst other WAN stats). 
Unfortunately this isn't exported as JMX or JSON automatically so I believe we can still improve the situation by aligning exposure of stats through diagnostics, JMX, JSON (and possibly other mechanisms)
